src/app/pages/market/ProductAction.php

```diff
@@ -100,7 +129,10 @@ public function __invoke($request, $response, $args)
       'openApiUrl' => $openApiUrl,
       'version' => $productVersion,
       'docUrl' => $docUrl,
-      'installNow' => $installNow
+      'installNow' => $installNow,
+      'versionsToDisplay' => $versionsToDisplay,
+      'switchVersion' => $showDevVersions ? "hide dev versions" : "show dev versions",
```

Comment:
I would vote for developer instead of just dev?!?

Reply:
eigentlich steht dev für development version?!